manifests/peer-validation.yaml

```diff
@@ -0,0 +1,152 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kilo
+---
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: "peer-validation.kilo.svc"
+webhooks:
+- name: "peer-validation.kilo.svc"
+  rules:
+  - apiGroups:   ["kilo.squat.ai"]
+    apiVersions: ["v1alpha1"]
+    operations:  ["CREATE","UPDATE"]
```

Comment:
Does our webhook need these permissions even though it is only validating and not mutating?